samples/client/petstore/swift5/validation/PetstoreClient/Classes/OpenAPIs/Configuration.swift

```diff
@@ -8,10 +8,6 @@ import Foundation
 
 open class Configuration {
     
-    // This value is used to configure the date formatter that is used to serialize dates into JSON format.
-    // You must set it prior to encoding any dates, and it will only be read once.
-    @available(*, unavailable, message: "To set a different date format, use CodableHelper.dateFormatter instead.")
-    public static var dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"
```

Comment:
I don't know why these changes are only in my branch, they appeared when I generated samples